test/e2e/setup.sh

```diff
@@ -25,6 +25,7 @@ export HUB_AGENT_IMAGE="${HUB_AGENT_IMAGE:-hub-agent}"
 export MEMBER_AGENT_IMAGE="${MEMBER_AGENT_IMAGE:-member-agent}"
 export REFRESH_TOKEN_IMAGE="${REFRESH_TOKEN_IMAGE:-refresh-token}"
 export PROPERTY_PROVIDER="${PROPERTY_PROVIDER:-azure}"
+export CLOUD_CONFIG_FILE="${CLOUD_CONFIG_FILE:-"/etc/kubernetes/provider/config.json"}"
```

Comment:
the default path should be the "azure_valid_config.yaml" under the test/e2e, right?

Reply:
azure_valid_config.yaml will be used to upload all the values like it does in line 191. These values will be imported into the secret as a file located in /etc/kubernetes/provider/config.json

Reply:
my bad, misunderstood this variable. thank you for the clarification.